source/TimePicker.js

```diff
@@ -609,17 +645,18 @@
 		/**
 		* @private
 		*/
-		updateHours: function (hour) {
+		updateValue: function (newHour, newMinute) {
 			var valueTime = this.value.getTime();
 
-			this.value.setHours(hour);
+			if (newHour !== undefined) { this.value.setHours(newHour); } 
```

Comment:
I think we can change this to be `if (newHour != null) this.value.setHours(newHour);`. Similarly, we can do this for the next line as well.